search/cities_boundaries_table.hpp

```diff
@@ -55,12 +55,6 @@ class CitiesBoundariesTable
       {
         for (auto const & p : boundary.m_bbox.Points())
           rect.Add(p);
```

Comment:
nit: adding Min() and Max() is enough
Alternatively, add bbox.ToRect() (since the points are created anyway) or add ForEachCorner() that would avoid creating a vector.

Reply:
Min(), Max() done